job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/XShellContext.groovy

```diff
@@ -0,0 +1,44 @@
+package javaposse.jobdsl.dsl.helpers.step
+
+import javaposse.jobdsl.dsl.AbstractContext
+import javaposse.jobdsl.dsl.JobManagement
+
+class XShellContext extends AbstractContext {
+    String commandLine
+    boolean executeFromWorkingDir = false
+    String regexToKill = null
+    int timeAllocated = 0
+
+    protected XShellContext(JobManagement jobManagement) {
+        super(jobManagement)
+    }
+
+    /**
+     * Adds xshell command.
+     */
+    void commandLine(String commandLine) {
+        this.commandLine = commandLine
+    }
+
+    /**
+     * Execute command from Working Dir
+     */
+    void executeFromWorkingDir(boolean executeFromWorkingDir) {
```

Comment:
The UI description for this is "Executable is in workspace dir", so "executeFromWorkingDir" is misleading. Rename this to "executableInWorkspaceDir". And set the parameters default value to true: `void executeFromWorkingDir(boolean executeFromWorkingDir = true) { ... }`